electron/dev/preload.ts

```diff
@@ -15,6 +15,7 @@ import {
 import { IronfishAccountManagerAction } from 'Types/IronfishManager/IIronfishAccountManager'
```

Comment:
thought: Not something we have to do now (or even soon), but I feel like `electron/dev` and `electron/prod` could be condensed into one file. I saw dev uses console.log, but we could write a logging function that logs to the console on dev, and logs to something like Sentry, LogRocket, etc on prod.

Not sure how feasible, but it would be nice not having to maintain two files.

Reply:
We can move it in separate task.
I think it would be good first issue for you.